app/src/main/res/layout/fragment_data.xml

```diff
@@ -1,132 +1,248 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-				xmlns:tools="http://schemas.android.com/tools"
-				android:id="@+id/fragment_data"
-				android:layout_width="match_parent"
-				android:layout_height="match_parent"
-				android:background="@color/background"
-				android:orientation="vertical">
-
-    <ImageView
-        android:id="@+id/fragment_data_logo"
-		android:layout_width="@dimen/logo_width"
-		android:layout_height="157dp"
-		android:layout_centerHorizontal="true"
-        android:layout_alignParentTop="true"
-		tools:src="@drawable/logo"
-        android:contentDescription="@string/app_name"
-        android:src="@drawable/logo"
-		android:layout_marginTop="@dimen/twenty_eight_dp"
-		/>
-
-    <LinearLayout
-        android:id="@+id/fragment_data_total_trees"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/fragment_data"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/lightGrey"
```

Comment:
Can we please have the background here match the color of the login screens?

Reply:
I was trying to match the UI on the issue description. Changed it to the same color as used in login screen now.